netlify.toml

```diff
@@ -21,12 +21,33 @@
 # Update to the latest version number
 # Enable permanent links to a release (link persists after release is archived)
 [[redirects]]
-  from = "/v0.13-docs/*"
+  from = "/v0.14-docs/*"
   to = "/docs/:splat"
   status = 200
 
 # Update to include the recently archived version
 # Redirects all previously published versions to the latest
+
+[[redirects]]
```

Comment:
This list of redirects are for versions which we remove from knative.dev. The last agreed number of versions to publish on the site was LATEST -3 so (.14, .13, .12, .11 should be published in the site). This is in case someone has a bookmark to an older version. Instead of a 404, we redirect them to the latest version (-> /docs/)

Therefore we should remove all but 0.10 from this list.

Reply:
Fine with me.  Go ahead.

Reply:
Done